htcanalyze/main.py

```diff
@@ -21,15 +21,22 @@
 
 import configargparse
 
+# import own classes
 from htcanalyze.htcanalyze import HTCAnalyze, raise_value_error
 from htcanalyze.logvalidator import LogValidator
+# import for colorful output
```

Comment:
This is also superfluous. If you don't know a package or module that is imported, you can read its documentation. From the use-cases in the code it should become obvious what the package contributes (otherwise a comment should be placed in the code, not in the import, because at the point of importing nothing is actually happening yet, most of the time).